src/chatty/util/api/UserInfoManager.java

```diff
@@ -80,6 +120,7 @@ public void resultReceived(Set<String> requested, Collection<UserInfo> result) {
             for (UserInfo info : result) {
                 notFound.remove(info.login);
                 perLogin.setResult(info.login, info);
+                perId.setResult(Long.valueOf(info.id), info);
```

Comment:
Doing this does not seem to have broken anything but I'd love some input on whether linking the 2 caches this way is an okay thing to do.